reference/docs-conceptual/whats-new/Migrating-from-Windows-PowerShell-51-to-PowerShell-7.md

```diff
@@ -0,0 +1,257 @@
+in the the Command Palette. This function switches VSCode into an ISE-style layout. The ISE-style
+layout give you all the new features and capabilities of PowerShell in a familiar user experience.
+
+To switch between the default layout and the new ISE layout, press;
+
++ <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd>
+
+Type and select: **PowerShell:Enable ISE Mode**
+
+To set the layout to the original VSCode layout, open the Command Palette, type and select
+**PowerShell: Disable ISE Mode (restore to defaults)**
+
+For details about customizing the VSCode layout to ISE, see
+[How to Replicate the ISE Experience in Visual Studio Code](/powershell/scripting/components/vscode/how-to-replicate-the-ise-experience-in-vscode)
+
+> [!NOTE]
+> The PowerShell Team and its partners are actively working to improve the scripting
+> experience in the PowerShell extension for VSCode. We currently have no plans to update the ISE
+> with new features. In the latest versions of Windows 10 and Windows Server, the ISE is now a
+> user-uninstallable feature. We have no plans to make the ISE unavailable in the future.
+
+## Profiles
+
+A PowerShell profile is a script that executes when PowerShell starts. You can use the profile as a
+logon script to customize the environment. You can add commands, aliases, functions, variables,
+snap-ins, modules, and PowerShell drives. You can also add other session-specific elements to your
+profile so they are available in every session without having to import or re-create them. Profiles
+don't exist until you create them.
+
+The path to the location of the profile has changed in PowerShell 7 and is reflected in the variable
+**$PROFILE**. In Windows PowerShell 5.1, the location of the path and the name of the profile is
+`$HOME\Documents\WindowsPowerShell\Microsoft.PowerShell_profile.ps1`.
+
+In PowerShell 7 the path has been changed to reflect the change in the product name from
+`Windows PowerShell` to `PowerShell` and is
+`$HOME\Documents\PowerShell\Microsoft.PowerShell_profile.ps1`.
+
+For more information about
+[Profiles](/powershell/module/microsoft.powershell.core/about/about_profiles?view=powershell-7).
+
+## PowerShell Remoting
+
+PowerShell remoting lets you run any PowerShell command on one or more remote computers. You can
+establish persistent connections, start interactive sessions, and run scripts on remote computers.
+
+### WS-Management remoting
+
+Windows PowerShell 5.1 and below use the `WS-Management` protocol for connection
+negotiation and data transport. If remoting has been
+enabled, installing PowerShell 7 will use the existing **WinRM** configuration.
+
+> [!NOTE]
+> PowerShell 7 will use the existing Windows PowerShell 5.1 endpoint for remoting
+> connections. To update PowerShell 7 to include it own endpoint, run the
+> `Install-PowerShellRemoting.ps1` located in **$PSHOME**. For information about connecting to
```

Comment:
We should tell people to run `Enable-PSRemoting` from within PS7 to setup the endpoint rather than using the script which was really specifically for Nano where you needed to setup the PSCore endpoint from WinPS